src/app/code/community/Shopgate/Framework/Model/Redirect/Observer.php

```diff
@@ -71,6 +71,6 @@ private function getRedirects()
         $js   = Mage::getModel('shopgate/redirect_type_js');
         $http = Mage::getModel('shopgate/redirect_type_http');
 
-        return $this->redirect->isTypeJavaScript() ? array($js) : array($http, $js);
+        return $this->redirect->isTypeJavascript() ? array($js) : array($http, $js);
```

Comment:
Good catch!

Reply:
I am so amazing even I surprise myself from day to day.
p.s. let's ignore the fact that I committed this error to begin with a while back ^_^